en/tutorials/tutorial_hello_sky.md

```diff
@@ -1,135 +1,177 @@
+    * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    * POSSIBILITY OF SUCH DAMAGE.
+    *
+    ****************************************************************************/
+
+   /**
+    * @file px4_simple_app.c
+    * Minimal application example for PX4 autopilot
+    *
+    * @author Example User <mail@example.com>
+    */
+
+   #include <px4_config.h>
+   #include <px4_tasks.h>
+   #include <px4_posix.h>
+   #include <unistd.h>
+   #include <stdio.h>
+   #include <poll.h>
+   #include <string.h>
+
+   #include <uORB/uORB.h>
+   #include <uORB/topics/sensor_combined.h>
+   #include <uORB/topics/vehicle_attitude.h>
+
+   __EXPORT int px4_simple_app_main(int argc, char *argv[]);
+
+   int px4_simple_app_main(int argc, char *argv[])
+   {
+       PX4_INFO("Hello Sky!");
+       return OK;
+   }
+   ```
+
+1. Create and open a new *cmake* definition file named **CMakeLists.txt**. 
+   Copy in the text below.
+   ```cmake
+   ############################################################################
+   #
+   #   Copyright (c) 2015 PX4 Development Team. All rights reserved.
+   #
+   # Redistribution and use in source and binary forms, with or without
+   # modification, are permitted provided that the following conditions
+   # are met:
+   #
+   # 1. Redistributions of source code must retain the above copyright
+   #    notice, this list of conditions and the following disclaimer.
+   # 2. Redistributions in binary form must reproduce the above copyright
+   #    notice, this list of conditions and the following disclaimer in
+   #    the documentation and/or other materials provided with the
+   #    distribution.
+   # 3. Neither the name PX4 nor the names of its contributors may be
+   #    used to endorse or promote products derived from this software
+   #    without specific prior written permission.
+   #
+   # THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+   # "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+   # LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+   # FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+   # COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+   # INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+   # BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+   # OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+   # AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+   # LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+   # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+   # POSSIBILITY OF SUCH DAMAGE.
+   #
+   ############################################################################
+   px4_add_module(
```

Comment:
Do we have a documented format on the cmake format for apps?
E.g. looking at this I think:
* `MODULE examples__px4_simple_app` - what is module command doing. What does the format of the string here?
* What should the stack size be set as?
* are additional sources on same line or separate lines
* What are DEPENDS, and specifically platforms__common. Are there any other "standard" depends.
* Does spacing, tabs, indentation matter?

Reply:
It's documented here: https://github.com/PX4/Firmware/blob/master/cmake/common/px4_base.cmake#L152